.github/workflows/build_and_test.yml

```diff
@@ -42,3 +42,32 @@ jobs:
             --packages-up-to obstacle_stop_planner_refine \
             --event-handlers console_cohesion+ \
             --return-code-on-test-failure
+
+  code-metrix:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repo
+        uses: actions/checkout@v2
+
+      - name: Measure code metrix
```

Comment:
Typo: metrix -> metrics. Same for others.

Reply:
Thanks😰 Fixed in b3ec0a8.